examples/booleans/booleans.bal

```diff
@@ -6,11 +6,11 @@ boolean flag = true;
 int n = flag ? 1 : 2;
 
 public function main() {
-    // Parentheses are optional in conditions, but curly braces are required in `if/else` and other compound statements.
+    // Parentheses are optional in conditions, 
```

Comment:
Break the comment into two lines because it will exceed the character limit of 100.